apis/clusterresources/v1beta1/awsencryptionkey_webhook.go

```diff
@@ -26,6 +26,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 
 	"github.com/instaclustr/operator/pkg/models"
+	"github.com/instaclustr/operator/pkg/utils/zerofieldvalidator"
```

Comment:
fieldsvalidator or reqfieldsvalidator maybe?

Reply:
agree, renamed to requiredfieldsvalidator